web/src/elements/forms/SearchSelect/tests/ak-search-select-view.test.ts

```diff
@@ -93,12 +93,14 @@ describe("Search select: Test Input Field", () => {
     });
 
     afterEach(async () => {
-        await document.body.querySelector("#a-separate-component")?.remove();
-        await document.body.querySelector("ak-search-select-view")?.remove();
-        // @ts-expect-error expression of type '"_$litPart$"' is added by Lit
-        if (document.body["_$litPart$"]) {
+        await browser.execute(() => {
```

Comment:
Found in the course of testing: these need to be executed in the context of the browser, which can then be `await()`ed; otherwise, they may not get cleared correctly, which may result in misleading test results.